app/controllers/v0/profile/connected_applications_controller.rb

```diff
@@ -8,12 +10,30 @@ def index
       end
 
       def destroy
-        app = OktaRedis::App.with_id(connected_accounts_params[:id])
-        app.user = @current_user
+        icn = @current_user.icn
+        client_id = :id
```

Comment:
Should this be `connected_accounts_params[:id]`?

Reply:
":id" is the only one that seemed to work when doing testing with different "ids". Tried connected_accounts_params[:id] but it failed the build, shown in the commit below:
https://github.com/department-of-veterans-affairs/vets-api/pull/13711/commits/346da3b63b42ec753c94da586ca7b08e7386d224